client/src/views/Report/Sunburst/index.js

```diff
@@ -0,0 +1,223 @@
+import React, { useMemo, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    PieChart,
+    Pie,
+    Cell,
+    Tooltip,
+    ResponsiveContainer,
+} from 'recharts';
+
+import styles from './styles.scss';
+
+const getSunBurstData = (dataObject) => {
+    const centerPieData = [{
+        key: dataObject.key,
+        name: dataObject.name,
+        size: dataObject.size,
+    }];
+
+    const firstLevelData = [];
+    const secondLevelData = [];
+
+    dataObject.children.forEach(({
+        key,
+        name,
+        size,
+        children,
+    }) => {
+        firstLevelData.push({
+            key,
+            name,
+            size,
+        });
+
+        secondLevelData.push(children.map(
+            child => ({
+                ...child,
+                firstLevelKey: key,
+            }),
+        ));
+    });
+
+    return {
+        centerPieData,
+        firstLevelData,
+        secondLevelData: secondLevelData.flat(),
+    };
+};
+
+const getFilteredData = (newlevel, newKey, dataValue) => {
+    if (newlevel === 'first') {
+        const filteredata = dataValue.children.filter(dvc => dvc.key === newKey);
```

Comment:
```suggestion
        const filteredData = dataValue.children.filter(dvc => dvc.key === newKey);
```

Reply:
We will be using '#rscz/SunBurst'. So sunburst removed !!